src/main/java/org/folio/circulation/domain/policy/RequestPolicy.java

```diff
@@ -60,20 +59,33 @@ public boolean allowsType(RequestType type) {
     return false;
   }
 
-  private static Map<RequestType, Set<UUID>> extractAllowedServicePoints(
+  public boolean allowsServicePoint(RequestType requestType, String servicePointId) {
+    log.debug("allowsPickupServicePoint:: parameters: requestType: {}, servicePointId: {}",
```

Comment:
Wrong method name. BTW, shouldn't it be more descriptive? Like, servicePointIsAllowedForRequestType or something similar?

Reply:
"servicePointIsAllowed" is pretty much the same as "allowsServicePoint", and request type is specified as one of the parameters so I'm not sure if it makes sense to also mention it in the method name